integration-tests/src/test/java/tachyon/worker/BlockServiceHandlerIntegrationTest.java

```diff
@@ -216,7 +219,8 @@ public void evictionTest() throws Exception {
 
     // File 3 should be in memory and one of file 1 or 2 should be in memory
     Assert.assertEquals(100, fileInfo3.inMemoryPercentage);
-    Assert.assertTrue(fileInfo1.inMemoryPercentage == 100 ^ fileInfo2.inMemoryPercentage == 100);
+    Assert.assertTrue("Exactly one of file1 and file2 should be 100% in memory",
+        fileInfo1.inMemoryPercentage == 100 ^ fileInfo2.inMemoryPercentage == 100);
```

Comment:
We don't want both conditions to be true, so we need either xor or a much more complicated expression